selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestGitHubServiceClient.java

```diff
@@ -187,6 +192,8 @@ public void deleteAllGrants(final String username, final String password) throws
           .setAuthorizationHeader(createBasicAuthHeader(username, password))
           .request();
     }
+
+    LOG.debug(format("Application grants '%s' were removed from github.com", grandsId));
```

Comment:
https://www.slf4j.org/faq.html#logging_performance

Reply:
Did you mean usage of parameterized messages?

Reply:
yep